### PR TITLE
Fix Druid metadata refresh.

### PR DESCRIPTION
**Pull request by coleslaw**
Could you take a look @mistercrunch, in particular `dashed/models.py` ?

### CODE REVIEW

File: dashed/models.py [hunk 1]
```diff
@@ -840,8 +840,12 @@ def latest_metadata(self):
             return
         max_time = results[0]['result']['maxTime']
         max_time = parse(max_time)
-        intervals = (max_time - timedelta(seconds=1)).isoformat() + '/'
-        intervals += (max_time + timedelta(seconds=1)).isoformat()
+        # Query segmentMetadata for 7 days back. However, due to a bug,
```
Comment (coleslaw) [inline dashed/models.py]: @mistercrunch I have to query [-7 day, -1 day] range because of a bug in 0.8.0. (see comment).

**Comment by coleslaw** [comment]
It's ready to be reviewed @mistercrunch @dayzzz

File: dashed/models.py (hunk 1 above)
Comment (hongbozeng) [inline dashed/models.py]: why we need to query for 7 days? even 10 minutes is good enough

Comment (coleslaw) [inline dashed/models.py]: Max has a point that some segment might be outside of the range.  -7 is a heuristic to get at least one segment.

**Comment by mistercrunch** [comment]
LGTM ! merging.

**Pull request merged**